plugin/db/driver.go

```diff
@@ -273,7 +278,8 @@ func ParseMigrationInfo(filePath, filePathTemplate string) (*MigrationInfo, erro
 		return nil, errors.Errorf("invalid file path template: %q", filePathTemplate)
 	}
 	if !myRegex.MatchString(filePath) {
-		return nil, errors.Errorf("file path %q does not match file path template %q", filePath, filePathTemplate)
+		// File path does not match file path template.
```

Comment:
What's the specific reason for this change?

This is normally error-prone because caller would normally write

mi, err := ParseMigrationInfo
if err != nil {
}

// use mi

Reply:
Yes, I’ve updated all the callers. The callers need to check the nil pointer in these cases.

This is to match parseSchemaFileInfo(); the parse() should return nil instead of an error when the input doesn't match the template.

https://github.com/bytebase/bytebase/blob/main/server/webhook.go#L730